dev-portal/src/pages/Admin/Admin.jsx

```diff
@@ -2,7 +2,12 @@ import React, { Component } from 'react'
 import { BrowserRouter as Router } from 'react-router-dom'
 
 import { ApiManagement, SideNav } from './'
-import { AdminRoute } from './../../';
+import { AdminRoute } from './../../'
```

Comment:
That import is gross.

Reply:
I've changed this to import from `'index'`.